mantidproject/mantid
# GudrunOutputLoader

**Description of work.**
* The first iteration of the `LoadGudrunOutput` algorithm allows Gudrun output data to be loaded into mantid.
* The algorithm allow loading of:
  * `.dcs01`  
  * `.mdcs01`
  * `.mint01` 
  * `.mdor01` 
  * `.mgor01`


**Report to:** [Helen Playford]

**To test:**

* locate the new testing files in the `build/ExternalData/Testing/Data/UnitTest` directory
  * `POLARIS00097947-min.{dcs01/mdcs01/mint01/mdor01/mgor01}`
* Try to load in each of these data sets using the `LoadGudrunOutput` algorithm.
* Check that each workspace loads okay and has 100 bins.
* Check that each workspace has only 1 spectra (except the `.dcs01` file which should have 5. 


*Note: These workspaces have been shortened to 100 bins to reduce execution time of unit tests hence the cutoff of the data may seem quite sharp as they originally held 2000 bins. also data may be "un-interesting" for the same reason*

Fixes #23298 

[Release notes](https://github.com/mantidproject/mantid/blob/9cb846f91b2b6fed42b7d35fe27750142cd30ad5/docs/source/release/v3.14.0/diffraction.rst)

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
